app/javascript/__tests__/modules/listingDetails/__snapshots__/ListingDetailsEligibility.test.tsx.snap

```diff
@@ -1334,19 +1314,15 @@ exports[`ListingDetailsEligibility displays listing details eligibility section
           className="expandable-text text-sm text-gray-700"
         >
            
-          <div>
-            <p>
-              Provide minimum of 4 years rental history with at least two prior rentals in which you were responsible for paying the rent.  Applicants without rental history will still be considered. No Guarantors permitted. 
-            </p>
-            <p>
-              Previous rental history will be reviewed and must exhibit no derogatory references. 
-            </p>
-            <p>
-              Landlord references will only check for...
-            </p>
-          </div>
+          <span>
+            Provide minimum of 4 years rental history with at least two prior rentals in which you were responsible for paying the rent.  Applicants without rental history will still be considered. No Guarantors permitted. 
+
+Previous rental history will be reviewed and must exhibit no derogatory references. 
+
+Landlord references will only check for...
+          </span>
```

Comment:
previously we were supporting markup here, e.g. see Additional Eligibility Rules on this listing https://dahlia-full.herokuapp.com/listings/a0W0P00000GwGl3UAF?react=true 

we are now stripping out the markup and putting the content in a `span`. is that expected? can we still preserve the markup/formatting?

<img width="1608" alt="image" src="https://user-images.githubusercontent.com/98352600/198114832-43d77bea-2fcc-4ff0-8130-105512c9772a.png">

Reply:
I took a look at the updated review app. It looks like we're visually retaining the whitespace now. Thanks! 

However I did notice that we're using lots of `span` tags now rather than `p`. Do we have any concerns semantically?

![image](https://user-images.githubusercontent.com/98352600/199117222-d5754d18-38ab-481d-a539-3b4f23668bdd.png)